packages/toolpad-app/next.config.mjs

```diff
@@ -79,94 +80,123 @@ const securityHeaders = [
   },
 ];
 
+const sentryWebpackPluginOptions = {
+  // Additional config options for the Sentry Webpack plugin. Keep in mind that
+  // the following options are set automatically, and overriding them is not
+  // recommended:
+  //   release, url, org, project, authToken, configFile, stripPrefix,
+  //   urlPrefix, include, ignore
+
+  silent: true, // Suppresses all logs
+
+  dryRun: !process.env.TOOLPAD_SENTRY_DSN,
```

Comment:
Hmm, I'm not sure this would work well for us. This only gets executed at build time. Would it still work if we don't upload sourcemaps and instead rely on production sourcemap urls? https://nextjs.org/docs/advanced-features/source-maps#configuration-flag

Reply:
Which thing in specific would not work well?
I added this `dryRun` condition because if `dryRun` has its default value Sentry seems to require an auth token, but an auth token should not be required if someone is not using Sentry.
This logic seems to fix that issue both locally and in production.

Reply:
i could add a comment making it more clear though if that's the confusion

Reply:
if it's about `sourceMaps` and there's a chance they could still be visible in the browser, we can try not uploading them and setting them manually in Sentry just for safety

Reply:
I don't mind them being visible in the browser. This is an OS project after all 🙂 .
My point is basically, if we have sourcemaps publicly available, hosted from the toolpad application, do we still need to upload them for Sentry? Will Sentry find the sourcemaps if our js files have `//# sourceMappingURL` directives? If yes, then I'd just disable uploading them altogether.

Reply:
Seems that we should be fine having soruce-maps hosted with app itself: https://docs.sentry.io/platforms/javascript/sourcemaps/#2-provide-source-maps-to-sentry